### ISSUE TITLE
Passing a transform function to 'reflect:' in a property descriptor has no effect without also specifying an observer

### ISSUE THREAD

**Issue by nathawes** [hybridsjs/hybrids#271]
For example, the two property descriptors below have the same value and reflect transform functions:
```
define({
    tag: 'issue-reproducer',
    reflected: {
        value: 'abc',
        reflect: value => value.toUpperCase()
    },
    reflectedAndObserved: {
        value: 'abc',
        reflect: value => value.toUpperCase(),
        observe: () => console.log('change observed')
    }
});
```
but their attribute values are reflected differently:
```
<issue-reproducer reflected="abc" reflected-and-observed="ABC"></issue-reproducer>
```

I think the call to `reflect` [here](https://github.com/hybridsjs/hybrids/blob/55919212616f8c22919bd4b361cce12b1166c482/src/value.js#L52) might just need to be changed to `fn`.

**Comment by smalluban** [comment]
Thanks for the issue. I'll check it as soon as possible.

**Comment by smalluban** [comment]
Fixed in cce3efde417849af9cdda557f9595adeeda75a7b

Thanks!
